packages/expo-asset/src/Asset.js

```diff
@@ -202,28 +261,41 @@ export default class Asset {
       return;
     }
     this.downloading = true;
+
     try {
-      const localUri = `${FS.cacheDirectory}ExponentAsset-${this.hash}.${this.type}`;
-      let exists, md5;
-      ({ exists, md5 } = await FS.getInfoAsync(localUri, {
-        cache: true,
-        md5: true,
-      }));
-      if (!exists || md5 !== this.hash) {
-        ({ md5 } = await FS.downloadAsync(this.uri, localUri, {
+      if (Platform.OS === 'web') {
+        if (isImageType(this.type)) {
+          const { width, height, name } = await getImageInfoAsync(this.uri);
+          this.width = width;
+          this.height = height;
+          this.name = name;
+        } else {
+          this.name = filenameFromUrl(this.uri);
+        }
+        this.localUri = this.uri;
+      } else {
+        const localUri = `${FS.cacheDirectory}ExponentAsset-${this.hash}.${this.type}`;
+        let exists, md5;
```

Comment:
Write out one statement per variable
```suggestion
        let exists;
        let md5;
```